style.css

```diff
@@ -130,8 +133,18 @@ body {
 	border-bottom: 1px solid var(--dark-border-color);
 }
 
-
 /* Centers some stuff*/
 .ng-table table th{
     vertical-align: middle;
 }
+
+/* Scroll buttons style*/
+.scroll-btns-grp  {
+  position: fixed; /* Fixed position */
+  bottom: 10px; /* Place the button at the bottom of the page */
+  right: 20px; /* Place the button 20px from the right */
+}
+
+.scroll-btns-grp .btn{
+    color: #000;
```

Comment:
I prefer to leave it at their default color instead of overriding them 🤔

Reply:
You prefer the buttons being white? I think they look better in black.

Reply:
Hmm, well we're not exactly UI/UX experts 😆 Its alright, this will do 👍

Reply:
Hmm, well we're not exactly UI/UX experts 😆 Its alright, this will do 👍